README.md

```diff
@@ -8,8 +8,9 @@
 
```

Comment:
- [ ] Kindly add a project description to the README, so your documentation is more professional